src/modules/Telemetry/EnvironmentTelemetry.cpp

```diff
@@ -151,48 +151,49 @@ uint32_t GetTimeSinceMeshPacket(const meshtastic_MeshPacket *mp)
-        display->setFont(FONT_SMALL);
-        display->drawString(x, y += fontHeight(FONT_MEDIUM), "Measurement Error");
+        display->drawString(x, y, "Measurement Error");
         LOG_ERROR("Unable to decode last packet");
         return;
     }
 
-    display->setFont(FONT_SMALL);
+    // Display "From: ..." on its own
+    display->drawString(x, y, "From: " + String(lastSender) + "(" + String(agoSecs) + "s)");
+
     String last_temp = String(lastMeasurement.variant.environment_metrics.temperature, 0) + "°C";
     if (moduleConfig.telemetry.environment_display_fahrenheit) {
         last_temp = String(CelsiusToFahrenheit(lastMeasurement.variant.environment_metrics.temperature), 0) + "°F";
     }
-    display->drawString(x, y += fontHeight(FONT_MEDIUM) - 2, "From: " + String(lastSender) + "(" + String(agoSecs) + "s)");
-    display->drawString(x, y += fontHeight(FONT_SMALL) - 2,
-                        "Temp/Hum: " + last_temp + " / " +
-                            String(lastMeasurement.variant.environment_metrics.relative_humidity, 0) + "%");
-    if (lastMeasurement.variant.environment_metrics.barometric_pressure != 0)
-        display->drawString(x, y += fontHeight(FONT_SMALL),
-                            "Press: " + String(lastMeasurement.variant.environment_metrics.barometric_pressure, 0) + "hPA");
-    if (lastMeasurement.variant.environment_metrics.voltage != 0)
-        display->drawString(x, y += fontHeight(FONT_SMALL),
-                            "Volt/Cur: " + String(lastMeasurement.variant.environment_metrics.voltage, 0) + "V / " +
-                                String(lastMeasurement.variant.environment_metrics.current, 0) + "mA");
-    if (lastMeasurement.variant.environment_metrics.iaq != 0)
+
+    // Continue with the remaining details
+    display->drawString(x, y += fontHeight(FONT_SMALL), "Temp/Hum: " + last_temp + " / " + String(lastMeasurement.variant.environment_metrics.relative_humidity, 0) + "%");
+
+    if (lastMeasurement.variant.environment_metrics.barometric_pressure != 0) {
+        display->drawString(x, y += fontHeight(FONT_SMALL), "Press: " + String(lastMeasurement.variant.environment_metrics.barometric_pressure, 0) + "hPA");
+    }
+
+    if (lastMeasurement.variant.environment_metrics.voltage != 0) {
+        display->drawString(x, y += fontHeight(FONT_SMALL), "Volt/Cur: " + String(lastMeasurement.variant.environment_metrics.voltage, 0) + "V / " + String(lastMeasurement.variant.environment_metrics.current, 0) + "mA");
+    }
+
+    if (lastMeasurement.variant.environment_metrics.iaq != 0) {
         display->drawString(x, y += fontHeight(FONT_SMALL), "IAQ: " + String(lastMeasurement.variant.environment_metrics.iaq));
-    if (lastMeasurement.variant.environment_metrics.distance != 0)
-        display->drawString(x, y += fontHeight(FONT_SMALL),
-                            "Water Level: " + String(lastMeasurement.variant.environment_metrics.distance, 0) + "mm");
+    }
```

Comment:
Please don't drop the distance display from the radar sensor.

Reply:
Sorry didn't notice. I put it back and reorganized lines as they used to keep it neat